bots/happinessncheer.bot.js

```diff
@@ -0,0 +1,48 @@
+const Slack = require('pico-slack');
+const cron = require('node-schedule');
+const {differenceInCalendarDays} = require('date-fns');
+
+const peeps = [
+	`Rbb`,
+	`Christian`,
+	`Evelyn`,
+	`LP`,
+	`Scott`,
+	`Rachel`,
+	`Tina`,
+	`David`,
+	`Katie`,
+	`Meg`,
+	`Greg`,
+	`Kellen`,
+	`Cathleen`,
+	`Chris`,
+	`Simon`,
+	`Carly`,
+	`Jared`,
+	`Mark`,
+	`Jenny`,
+	`Conrad`,
+	`Ross`,
+	`Ryan`,
+];
```

Comment:
This list is also missing Christie and Sarah.

Reply:
Perfect, I just grabbed a list for a random pinned doc as it's the only thing I could find. Do you know where they sit in the order?

Reply:
It's the same doc, but you have to scroll to the bottom for the latest iteration. The order you have here is perfect except for Christie and Sarah at the end.